### ISSUE TITLE
Icon Request: fa-letsencrypt

### ISSUE THREAD

**Issue by noxifoxi** [FortAwesome/Font-Awesome#6660]
Let's Encrypt is an upcoming service for free TLS (HTTPS) certificates for everyone.

> Let’s Encrypt is a new Certificate Authority: 
> It’s free, automated, and open. 
> Arriving Mid-2015

https://letsencrypt.org/

**Comment by sebie** [comment]
To be honest, I would wait till it actually comes out. But, yes it would be nice for @letsencrypt , @letsencrypt/admin to have a badge or FontAwesome icon, to represent the use of it.

Idea's | Thoughts?

**Comment by zavaboy** [comment]
I support this! I'd use fa-letsencrypt on my encrypted sites to support the cause. I'm getting excited; they are [launching in less than a month](https://letsencrypt.org/2015/08/07/updated-lets-encrypt-launch-schedule.html) with general availability!

The logo used on the top of [their secured demo page](https://helloworld.letsencrypt.org/) is an SVG, which may be a useful asset:
![Let's Encrypt logo SVG](https://helloworld.letsencrypt.org/images/LE_badge_color.svg)

**Comment by kevdotbadger** [comment]
It's out in public beta, so now might be a chance to get this added? I know a lot of developers who have already launched SSL's from Let's Encrypt and are keen to show support.

**Comment by mizerydearia** [comment]
+1

**Comment by kwendenarmo** [comment]
+1

**Comment by ianklemm** [comment]
+1

**Comment by realulim** [comment]
Letsencrypt is quickly becoming a de-facto standard, so ++1

**Comment by prae-torian** [comment]
+1

**Comment by Aerendir** [comment]
Any news on this? When will it be added?

**Comment by zsoltsandor** [comment]
Huge +1.

**Comment by jenstroeger** [comment]
Yep, +1 and it’s April 202. Let’s Encrypt has grown big and a brand icon would be great for websites! Their logo hasn’t changed:

![Let’s Encrypt logo](https://letsencrypt.org/images/letsencrypt-logo-horizontal.svg)

**Comment by hpvd** [comment]
+1

**Comment by Manawyrm** [comment]
+1 - I would've liked to use this right now. Unfortunately it does not exist yet :frowning_face:

**Comment by PerseusTheGreat** [comment]
Is there any plan to create/present it?
7 years past...

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.

**Comment by PerseusTheGreat** [comment]
@sensibleworld 
done! #21062
